.github/workflows/devRun.yml

```
@@ -22,26 +22,24 @@ jobs:
           virtualenvs-create: true
           virtualenvs-in-project: true
           installer-parallel: true
-          virtualenvs-path: .venv
+      - name: Load cached virtual environment
+        id: cache-venv
+        uses: actions/cache@v4
+        with:
+          path: .venv
+          key: venv-${{ runner.os }}-${{ steps.setup-python.outputs.python-version }}-${{ hashFiles('**/poetry.lock') }}
```

actionlint (1.7.4)

property "setup-python" is not defined in object type {}

(expression)